control/control.rnc

```diff
@@ -1057,6 +1057,29 @@ update = element update {
 
 # update
 
+configuration_management = element configuration_management {
+    element type { "salt" | "puppet" }? &
```

Comment:
Is the type really optional?

Reply:
You are right. It should not be optional.